src/System.Console/tests/Color.cs

```diff
@@ -92,6 +92,68 @@ public static void NonRedirectedOutputDoesUseAnsiSequences()
         Assert.Equal(3, Encoding.UTF8.GetString(data.ToArray()).ToCharArray().Count(c => c == Esc));
     }
 
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            // Skip directories that dont exist
+            if (Directory.Exists(location) == false)
```

Comment:
We don't have a style guideline on it as far as I'm aware.  I personally prefer the ```!Directory.Exists(location)``` approach because it reads better to me, i.e. "if not exists" rather than "if exists is false".  Not a big deal.